### DISCUSSION TITLE
参考文献可以用[1]这样上标吗？

### DISCUSSION THREAD

**Discussion by earthmoon** [whutug/whu-thesis#75] [General]
因为我看武大的要求是要上标的，但是我看模板没有上标。

**Comment by sikouhjw** [comment]
https://github.com/whutug/whu-thesis/blob/a0cb5918638fece6c7fa171bb6be179dcf6a3a33/whu-thesis-doc.tex#L477

很显然，模板做了处理。至于是不是与规范一致，请你提供论证。

**Reply by earthmoon** [reply]
> https://github.com/whutug/whu-thesis/blob/a0cb5918638fece6c7fa171bb6be179dcf6a3a33/whu-thesis-doc.tex#L477
> 
> 很显然，模板做了处理。至于是不是与规范一致，请你提供论证。

我看了，是一致的。抱歉了。

**Reply by earthmoon** [reply]
> https://github.com/whutug/whu-thesis/blob/a0cb5918638fece6c7fa171bb6be179dcf6a3a33/whu-thesis-doc.tex#L477
> 
> 很显然，模板做了处理。至于是不是与规范一致，请你提供论证。

你好，国标中要求文献的引用在[数字]之后加上页码，而文后的文献目录不再有页码，和我们的是不是有区别啊

**Reply by sikouhjw** [reply]
> > https://github.com/whutug/whu-thesis/blob/a0cb5918638fece6c7fa171bb6be179dcf6a3a33/whu-thesis-doc.tex#L477
> > 
> > 很显然，模板做了处理。至于是不是与规范一致，请你提供论证。
> 
> 你好，国标中要求文献的引用在[数字]之后加上页码，而文后的文献目录不再有页码，和我们的是不是有区别啊

你给个国标截图

**Reply by earthmoon** [reply]
> > > https://github.com/whutug/whu-thesis/blob/a0cb5918638fece6c7fa171bb6be179dcf6a3a33/whu-thesis-doc.tex#L477
> > > 
> > > 很显然，模板做了处理。至于是不是与规范一致，请你提供论证。
> > 
> > 
> > 你好，国标中要求文献的引用在[数字]之后加上页码，而文后的文献目录不再有页码，和我们的是不是有区别啊
> 
> 你给个国标截图

![image](https://user-images.githubusercontent.com/45897471/124115680-adc62a80-daa0-11eb-8db5-f60fc94e9b38.png)

**Reply by sikouhjw** [reply]
> > > > https://github.com/whutug/whu-thesis/blob/a0cb5918638fece6c7fa171bb6be179dcf6a3a33/whu-thesis-doc.tex#L477
> > > > 
> > > > 很显然，模板做了处理。至于是不是与规范一致，请你提供论证。
> > > 
> > > 
> > > 你好，国标中要求文献的引用在[数字]之后加上页码，而文后的文献目录不再有页码，和我们的是不是有区别啊
> > 
> > 
> > 你给个国标截图
> 
> ![image](https://user-images.githubusercontent.com/45897471/124115680-adc62a80-daa0-11eb-8db5-f60fc94e9b38.png)

bibtex 应该做不到，去看看 `texdoc biblatex-gb7714-2015`，用 `biblatex` 应该能做到

**Reply by earthmoon** [reply]
> > > > > https://github.com/whutug/whu-thesis/blob/a0cb5918638fece6c7fa171bb6be179dcf6a3a33/whu-thesis-doc.tex#L477
> > > > > 
> > > > > 很显然，模板做了处理。至于是不是与规范一致，请你提供论证。
> > > > 
> > > > 
> > > > 你好，国标中要求文献的引用在[数字]之后加上页码，而文后的文献目录不再有页码，和我们的是不是有区别啊
> > > 
> > > 
> > > 你给个国标截图
> > 
> > 
> > ![image](https://user-images.githubusercontent.com/45897471/124115680-adc62a80-daa0-11eb-8db5-f60fc94e9b38.png)
> 
> bibtex 应该做不到，去看看 `texdoc biblatex-gb7714-2015`，用 `biblatex` 应该能做到

文档中明确说明做不到，并且如果只出现一次，那么这个页码要出现在**参考文献**目录中。比较智能了，目前好像无解。

**Reply by zepinglee** [reply]
> bibtex 应该做不到，去看看 `texdoc biblatex-gb7714-2015`，用 `biblatex` 应该能做到

`\cite[page]{key}`

**Comment by gongwen1205** [comment]
请教下为啥我设置了cite-style={numerical-super}还是author-year形式呢？

**Reply by gongwen1205** [reply]
> 请教下为啥我设置了cite-style={numerical-super}还是author-year形式呢？

![image](https://user-images.githubusercontent.com/62161547/157676167-bbfa9a4c-5bc4-42e8-b6e1-2a1d1cc319ec.png)

**Reply by xkwxdyy** [reply]
> 请教下为啥我设置了cite-style={numerical-super}还是author-year形式呢？

应该会有一个冲突的警告
![image](https://user-images.githubusercontent.com/87134063/157680794-c82c3d0e-a18b-44c1-80af-468acf17b4b1.png)

**Reply by xkwxdyy** [reply]
请一定先确认好`bib-style`和`cite-style`的样式

**Reply by gongwen1205** [reply]
ok非常感谢你~

**Reply by xkwxdyy** [reply]
如果你们学院的要求确实是这样的话，就请提一个issue

**Reply by gongwen1205** [reply]
学院没有特别要求但在翻硕士学位论文的时候发现都是上标引用形式，这是我随便找到一篇学位论文http://paperright.lib.whu.edu.cn/read/pdfindex.jsp?fid=53379efa1424b5ad42b96b080007d203

**Reply by xkwxdyy** [reply]
那可以考虑使用`biblatex`方式 用`biber`编译
